### PR TITLE
Feature/KAK-79-switch-between-iframes

### PR DESCRIPTION
**Pull request by konrad-wywiol**

### CODE REVIEW

File: src/pages/base.ts [hunk 1]
```diff
@@ -86,6 +86,13 @@ class Page {
     return browser.executeScript('arguments[0].scrollIntoView(false);', this.getElement(elementName).getWebElement());
   }
 
+  public switchIframe(elementName) {
```
Comment (tgorskievo) [inline src/pages/base.ts]: You can add type here `(elementName: string)`

File: readme.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <a href="https://travis-ci.org/TheSoftwareHouse/Kakunin"><img src="https://travis-ci.org/TheSoftwareHouse/Kakunin.svg?branch=master" alt="build status" height="18"></a>
   &emsp;
   <img src="/data/npm.png" alt="npm.png" width="25"/> 
-  Current npm version:
+  Current npm version:
```
Comment (joszo) [inline readme.md]: unnecessary space

File: src/pages/base.ts [hunk 3]
```diff
@@ -24,11 +24,11 @@ class Page {
     const url =
       data.raw().reduce((prev, item) => {
         if (prev.indexOf(`:${item[0]}`) === -1) {
-          additionalParams.push(item);
+          additionalParams[item[0]] = item[1];
```
Comment (sethii) [inline src/pages/base.ts]: What is a reason for this change?

Comment (joszo) [reply src/pages/base.ts]: It was changed in @tgorskievo PR https://github.com/TheSoftwareHouse/Kakunin/pull/211/files

Comment (sethii) [reply src/pages/base.ts]: Ok I know why

Comment (tgorskievo) [reply src/pages/base.ts]: Did you create this branch from a current `next`? 

I updated merged `master` to `next` in a separated PR which includes missing doc version `3.0.1`

**Pull request merged**